cacti/tests/compose/docker-compose.yaml

```diff
@@ -2,11 +2,43 @@ version: "3"
 
 services:
   cacti:
-    build: .
-    container_name: dd-test-cacti
+    container_name: cacti
+    image: datadog/docker-library:cacti-1.2.19
+    domainname: example.com
+    hostname: cacti
     ports:
-      - "13306:3306"
       - "8080:80"
+    environment:
+      - DB_NAME=cacti_master
+      - DB_USER=cactiuser
+      - DB_PASS=cactipassword
+      - DB_HOST=db
+      - DB_PORT=3306
+      - DB_ROOT_PASS=rootpassword
+      - INITIALIZE_DB=1
+      - TZ=America/Los_Angeles
+    volumes:
+      - cacti-data:/cacti
+      - cacti-spine:/spine
+      - cacti-backups:/backups
+    links:
+      - db
+
+  db:
+    image: "mariadb:10.3"
+    container_name: cacti_db
+    domainname: example.com
+    hostname: db
+    ports:
+      - "13306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=rootpassword
+      - TZ=America/Los_Angeles
     volumes:
-      - ${RRD_PATH}:/var/www/html/cacti/rra
-      - ${DD_LOG_1}:/opt/cacti/log/cacti.log
+      - cacti-db:/var/lib/mysql
+
+volumes:
+  cacti-db:
+  cacti-data:
+  cacti-spine:
+  cacti-backups:
```

Comment:
Why do we need to create those volumes? Do we keep them between executions? 

If not, I think we can just remove them

Reply:
Locally we may need them to initialize database etc and make the boot process faster. IMHO, I think they don't bother us. Anyway, I removed them...

Reply:
I hadn't thought about the permission issues that mounting files or directories can cause instead of working with docker volumes. I have restored the volumes to avoid those problems.

Reply:
🤔 I don't understand why we have permissions issues during the e2e tests. Could you tell me what's wrong exactly here? I'm a bit confused 😕

Reply:
I hope [this](https://github.com/DataDog/integrations-core/runs/8469577494) helps you. Cacti needs permissions in some directories we mount (database, backups, etc). It can be different configured (permissions) locally or in CI. The best way to avoid that is configuring docker volumes and making it independent from the platform where it is running.

Reply:
Ah you forgot to remove the volumes in the image definitions, that's why it's trying to mount a local folder and you get the permission error. [This](https://github.com/DataDog/integrations-core/pull/12991/commits/fb8557131e718c1e1b3385a35c4c640ced37d727) will fix your issue. CI: https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=112865&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=57642e18-890a-5eef-b695-2b3c448c1261

Reply:
No. I didn't forget. I don't want to stop mounting those directories in case we have to pass some initialization or backup to the database from outside.

Reply:
Not sure that's useful to have them in the CI because they will never be useful in this case, and this is where the tests are mostly running. But ok let's keep them